drivers/SmartThings/philips-hue/src/disco/init.lua

```diff
@@ -18,19 +20,22 @@ local DOMAIN = "local"
 local HueDiscovery = {
   api_keys = {},
   disco_api_instances = {},
-  light_state_disco_cache = {},
+  device_state_disco_cache = {},
   ServiceType = SERVICE_TYPE,
   Domain = DOMAIN,
   discovery_active = false
 }
 
 local supported_resource_types = {
-  light = true
+  light = LightDiscovery.process_discovered_light
```

Comment:
Is the intention that each new device type will get its own `*Discovery` module with a function referenced here to process it once the driver becomes aware of the device?